### DISCUSSION TITLE
Optional instant flight paths

### DISCUSSION THREAD

**Discussion by krutoj87** [chromiecraft/chromiecraft#7286] [Suggestions]
On other servers I have seen, that they made flight paths instant with the option to disable it with an option at flight masters. I think this could be a really good quality of live addition for the people, which fly the same flight paths for 15 years or more now. Also it could increase participation in group activities, in more remote areas. To me it happened often, that I ignored requests for group quests in netherstorm, because I was in shadowmoon valley and didn't want to spend 20 minutes flying to netherstorm and back.
And if they are optional players could enable and disable them and experience the game however they like it.

In my opinion flight paths don't really serve a purpose other than wasting players time. Maybe it was different when everything was new and it still was a cool experience to fly over the zones. But we are talking about an old game here, so I think optional instant flight paths could be considered as quality of life option.

**Comment by armchairrockstar1** [comment]
I don't like this idea. It's extremely NOT blizzlike.

**Reply by krutoj87** [reply]
Cross faction also isn't blizzlike but we have it, because it's QoL and the whole idea of factions is pointless on PvE servers. Just like flight paths are pointless on any server. 

But I would also be fine if we could double the flightpath speed. Just anything that reduces the down times of flight paths, because some flight paths probably take up to 10 minutes.
